test/test_chatcommands.py

```diff
@@ -36,15 +36,15 @@ def test_coffee():
     msg = Fake({"owner": {"name": "El'endia Starman"}})
 
     coffees = "\\*brews a cup of ({}) for ".format("|".join(chatcommands.COFFEES))
-    assert regex.match(coffees + "@El'endiaStarman\\*", chatcommands.coffee(None, original_msg=msg))
+    assert regex.match(coffees + "El'endiaStarman\\*", chatcommands.coffee(None, original_msg=msg))
```

Comment:
Again, the `@` is required to ping the original requester of the command.  We use this sort of like a 'ping' comand to make sure Smokey is alive, but we do it in a more fun way.